es6.md

```diff
@@ -145,3 +155,37 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Declaration {
+    default: boolean;
+    declaration: VariableStatement | Declaration | HoistableDeclaration | null;
```

Comment:
@sebmck @caridy I forgot where we discussed adding the HoistableDeclaration node. Did @michaelficarra or @ariya have issues there? I forgot...